screwdriver.yaml

```diff
@@ -3,7 +3,11 @@ shared:
 
 jobs:
     main:
-        requires: [~pr, ~commit]
+        requires: [
```

Comment:
how about change it to this format 
```
- ~pr
-  ~commit
- ....
```?
That's easier to read

Reply:
added